core/src/main/java/org/opensearch/sql/analysis/symbol/Namespace.java

```diff
@@ -13,6 +13,7 @@ public enum Namespace {
 
   INDEX_NAME("Index"),
   FIELD_NAME("Field"),
+  HIDDEN_FIELD_NAME("HiddenField"),
```

Comment:
Maybe
```suggestion
  HIDDEN_FIELD_NAME("MetaField"),
```

Reply:
Hidden seems more cross-datasource to me.